.github/workflows/ci_tests_backward.yaml

```
@@ -0,0 +1,105 @@
+
+name: GMT Backward Tests
+
+on:
+  # push:
+  #   branches: [ main ]
+  pull_request:
+    # types: [ready_for_review]
+    paths-ignore:
+      - 'doc/**'
+      - 'examples/**'
+      - '*.md'
+      - 'README.rst'
+      - 'LICENSE.txt'
+      - '.gitignore'
+  # Schedule tests on Monday/Wednesday/Friday
+  schedule:
+    - cron: '0 0 * * 1,3,5'
+
+jobs:
+  test:
+    name: ${{ matrix.os }} - GMT ${{ matrix.gmt_version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.10']
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        gmt_version: ['6.3']
+    timeout-minutes: 30
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      # Cancel previous runs that are not completed
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
+
+      # Checkout current git repository
+      - name: Checkout
+        uses: actions/checkout@v3.0.2
+        with:
+          # fetch all history so that setuptools-scm works
+          fetch-depth: 0
+
+      # Install Mambaforge with conda-forge dependencies
+      - name: Setup Mambaforge
+        uses: conda-incubator/setup-miniconda@v2.1.1
+        with:
+          activate-environment: pygmt
+          python-version: ${{ matrix.python-version }}
+          channels: conda-forge,nodefaults
+          channel-priority: strict
+          miniforge-version: latest
+          miniforge-variant: Mambaforge
+          mamba-version: "*"
+          use-mamba: true
+
+      # Install GMT and other required dependencies from conda-forge
+      - name: Install dependencies
+        run: |
+          mamba install gmt=${{ matrix.gmt_version }} numpy \
+                        pandas xarray netCDF4 packaging geopandas \
+                        build dvc make pytest>=6.0 \
+                        pytest-cov pytest-doctestplus pytest-mpl sphinx-gallery
+
+      # Show installed pkg information for postmortem diagnostic
+      - name: List installed packages
+        run: mamba list
+
+      # Download cached remote files (artifacts) from GitHub
+      - name: Download remote data from GitHub
+        uses: dawidd6/action-download-artifact@v2.22.0
+        with:
+          workflow: cache_data.yaml
+          workflow_conclusion: success
+          name: gmt-cache
+          path: .gmt
+
+      # Move downloaded files to ~/.gmt directory and list them
+      - name: Move and list downloaded remote files
+        run: |
+          mkdir -p ~/.gmt
+          mv .gmt/* ~/.gmt
+          # Change modification times of the two files, so GMT won't refresh it
+          touch ~/.gmt/server/gmt_data_server.txt ~/.gmt/server/gmt_hash_server.txt
+          ls -lhR ~/.gmt
+
+      # Pull baseline image data from dvc remote (DAGsHub)
+      - name: Pull baseline image data from dvc remote
+        run: |
+          dvc pull --verbose
```

```suggestion
          dvc pull pygmt/tests/baseline/test_image.png --verbose
```
The dvc pull step is a common failure point, so best to only pull the one file needed.